src/fontManager.h

```diff
@@ -45,11 +45,18 @@ class FontManager {
 	nameColorsMap fonts;
 	
 public:
+	FontManager();
 	FontManager(Graphics* graphics);
+	FontManager(const FontManager& fm);	// copy constructor
+	//FontManager& operator=(const FontManager& rhs) {};
```

Comment:
Constructor for copying fontManager. You pass in an already initialized fontManager and create a copy of it. The copy does not need to reinitialize all the font textures and can already be used.